crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -64,7 +64,7 @@ pub struct DWalletMPCManager {
     pub(crate) malicious_actors: HashSet<AuthorityName>,
     weighted_threshold_access_structure: WeightedThresholdAccessStructure,
     weighted_parties: HashMap<PartyID, PartyID>,
-    outputs_manager: DWalletMPCOutputsVerifier,
+    pub outputs_manager: DWalletMPCOutputsVerifier,
```

Comment:
why should it be public? I wanted it to be internal only to detect malicious actors